### PR TITLE
added lenten (vegan) chili recipe (updated with picture)

### PR DESCRIPTION
**Pull request by yorgez**
One of my favorite recipes for Great Lent or any other fasting day.

### CODE REVIEW

File: src/lenten-chili.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Lenten (Vegan) Chili
+
+- Prep time: 15 Minutes
+- Cook time: 45 Minutes
+- Serves: 4-8 people
```
Comment (Knarkzel) [inline src/lenten-chili.md:5]: Use the emoji-variant:

```
- ⏲️ Prep time: 10 min
- 🍳Cook time: 30 min
- 🍽️ Servings: 4
```

File: src/lenten-chili.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+## Directions
+
+1. Chop the large onion and mince the garlic cloves.
+2. On medium-low heat in a large tall pot add 2-3 tbsp olive oil.
+3. Add the chopped onions into the pot with oil and constantly stir for 5-10 minutes until they begin to brown.
+4. Just before onions are done add the minced garlic to the pot and let cook for 1-2 minutes.
+5. Add the tomato sauce, diced tomatoes, sweet corn, beans, and rice.
+6. Turn up the heat to medium.
+6. Add each of the spices, taste the chili, and adjust flavoring accordingly.
+7. let cook for 20-30 minutes, stirring occasionally.
```
Comment (Knarkzel) [inline src/lenten-chili.md:33]: Capitalize `let`.

File: src/lenten-chili.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# Lenten (Vegan) Chili
+
```
Comment (Knarkzel) [inline src/lenten-chili.md:2]: Add `.webp` image if possible.

File: src/lenten-chili.md (hunk 1 above)
Comment (yorgez) [reply src/lenten-chili.md]: Thank you, I updated this and the other problem you pointed out and added an image.

**Pull request closed without merging**